cfme/tests/containers/test_containers_default_views.py

```diff
@@ -0,0 +1,268 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from cfme.web_ui import toolbar as tb, ButtonGroup, form_buttons
+from cfme.configure import settings
+from utils import testgen
+from utils.version import current_version
+
+
+pytestmark = [
+    pytest.mark.uncollectif(
+        lambda: current_version() < "5.6"),
+    pytest.mark.usefixtures('setup_provider'),
+    pytest.mark.tier(3)]
+pytest_generate_tests = testgen.generate(
+    testgen.container_providers, scope='function')
+
+
+# CMP-9936
+
+
+def test_containers_providers_view():
+    """
+
+    This test checks successful change of default views settings for Containers --> Providers menu
+    * Goes to Settings --> My Settings --> Default Views menu and change the default view
+     settings of Containers --> Containers Providers to Grid/Tile/List view
+    * Goes to Compute --> Containers --> Providers and verifies the selected view
+
+    """
+    # Change the view to Grid View
+    sel.force_navigate('my_settings_default_views')
+    bg = ButtonGroup('Containers Providers')
+    bg.choose('Grid View')
+    sel.click(form_buttons.save)
+
+    # Go to Container Providers and verify that the View changed to Grid View
+    sel.force_navigate('containers_providers')
+    assert tb.is_active('Grid View'), 'Providers grid view setting failed'
+
+    # Change the view to Tile View
+    sel.force_navigate('my_settings_default_views')
+    bg = ButtonGroup('Containers Providers')
+    bg.choose('Tile View')
+    sel.click(form_buttons.save)
+
+    # Go to Container Providers and verify that the View changed to Tile View
+    sel.force_navigate('containers_providers')
+    assert tb.is_active('Tile View'), 'Providers tile view setting failed'
+
+    # Change the view to List View
+    sel.force_navigate('my_settings_default_views')
+    bg = ButtonGroup('Containers Providers')
+    bg.choose('List View')
+    sel.click(form_buttons.save)
+
+    # Go to Container Providers and verify that the View changed to Tile View
+    sel.force_navigate('containers_providers')
+    assert tb.is_active('List View'), 'Providers list view setting failed'
+
+# CMP-9937
+
+
+def test_containers_projects_view():
+    """
+
+    This test checks successful change of default views settings for Containers --> Projects menu
+    * Goes to Settings --> My Settings --> Default Views menu and change the default view
+    settings of Containers --> Containers Projects to Grid/Tile/List view
+    * Goes to Compute --> Containers --> Projects and verifies the selected view
```

Comment:
Can you please apply the same styling as in the other test descirptions?

```python
"""Test description, rather short
# an empty line in between
Steps:
    * foo the bar
    * bar the baz!
```

You can also add a Prerequisities section if that is required, same style applies.